requirements-provider.txt

```diff
@@ -6,5 +6,6 @@ psycopg2-binary==2.8.4
 pymongo==3.10.1
 rasterio
 scipy
+tinydb
```

Comment:
requirements-processes.txt ?

Reply:
Good point.  Once OARec is implemented, TinyDB will also be the backend. At that point we can decide whether to move it into core requirements or not.